crates/ironrdp-pdu/src/rdp/client_info.rs

```diff
@@ -313,6 +313,9 @@ impl Encode for ExtendedClientOptionalInfo {
             dst.write_array(reconnect_cookie);
         }
 
+        dst.write_u16(0); // reserved1
+        dst.write_u16(0); // reserved2
```

Comment:
question: Did you fix a bug? Could you elaborate that? Please, open a separate PR dedicated to this with a `bug` commit type.